examples/website/3d-tiles/app.js

```diff
@@ -54,7 +54,7 @@ export class App extends PureComponent {
         // Update deck.gl viewState, moving the camera to the new tileset
         longitude: cartographicCenter[0],
         latitude: cartographicCenter[1],
-        zoom: zoom + 1.5, // TODO - remove adjustment when Tileset3D calculates correct zoom
+        zoom,
```

Comment:
is it corrected?

Reply:
Trying to make the model has a higher density, easier to perceive.